### PR TITLE
Fix typos detected by github.com/client9/misspell

### PR DESCRIPTION
**Pull request by seratch**
This pull request fixes minor typos detected by [misspell](https://github.com/client9/misspell). 

I've ignored the past changelogs and false positives. If you would like me to work on other files as well, let me know. Also, if the change on the method `rb_execarg_addopt` should not be included, I can revert the change (I judged the change is safe enough, though).

## before

```
 misspell .
configure.ac:2545:2: "nto" is a misspelling of "not"
configure.ac:2593:2: "nto" is a misspelling of "not"
doc/ChangeLog-0.60_to_1.1:541:3: "avaiable" is a misspelling of "available"
doc/ChangeLog-0.60_to_1.1:3576:20: "concatination" is a misspelling of "contamination"
doc/ChangeLog-2.1.0:16041:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.8.0:10128:29: "ASIGN" is a misspelling of "ASSIGN"
doc/ChangeLog-2.2.0:1005:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:8597:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11772:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11790:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11825:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11836:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11845:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11857:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11889:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11974:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11976:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11976:47: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11979:43: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11981:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12041:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12054:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12095:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12486:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12492:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12496:30: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12498:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12498:49: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12499:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12499:49: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12508:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:15154:51: "infinit" is a misspelling of "infinite"
doc/ChangeLog-2.0.0:15399:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:21274:29: "execption" is a misspelling of "exception"
doc/ChangeLog-YARV:1484:29: "separete" is a misspelling of "seperate"
doc/ChangeLog-YARV:1484:38: "assersions" is a misspelling of "assertions"
doc/ChangeLog-YARV:4976:45: "arugment" is a misspelling of "argument"
doc/ChangeLog-YARV:6325:11: "analisys" is a misspelling of "analysis"
doc/ChangeLog-YARV:6492:1: "redesgin" is a misspelling of "redesign"
doc/ChangeLog-2.4.0:891:51: "nonsence" is a misspelling of "nonsense"
doc/ChangeLog-2.4.0:9382:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.3.0:4294:51: "convertion" is a misspelling of "conversion"
doc/NEWS-1.9.2:243:36: "avaiable" is a misspelling of "available"
doc/NEWS-2.2.0:284:14: "addopt" is a misspelling of "adopt"
enc/Makefile.in:52:23: "DONIG" is a misspelling of "DOING"
enc/trans/big5-uao-tbl.rb:14114:4: "DAED" is a misspelling of "DEAD"
enc/trans/big5-hkscs-tbl.rb:10399:4: "DAED" is a misspelling of "DEAD"
enc/trans/big5-hkscs-tbl.rb:28817:11: "DAED" is a misspelling of "DEAD"
enc/trans/euckr-tbl.rb:4920:4: "DAED" is a misspelling of "DEAD"
enc/trans/gbk-tbl.rb:19254:4: "DAED" is a misspelling of "DEAD"
doc/ChangeLog-1.9.3:6960:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:7544:43: "embeded" is a misspelling of "embedded"
doc/ChangeLog-1.9.3:9566:37: "Assersions" is a misspelling of "Assertions"
doc/ChangeLog-1.9.3:9784:3: "funciton" is a misspelling of "function"
doc/ChangeLog-1.9.3:13493:39: "DONIG" is a misspelling of "DOING"
doc/ChangeLog-1.9.3:49252:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49252:46: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49267:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49276:19: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49280:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49282:43: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49290:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49293:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:62972:41: "prototye" is a misspelling of "prototype"
doc/ChangeLog-1.9.3:76349:31: "arguemnts" is a misspelling of "arguments"
doc/ChangeLog-1.9.3:77288:3: "Execption" is a misspelling of "Exception"
doc/ChangeLog-1.9.3:77438:17: "initalize" is a misspelling of "initialize"
doc/ChangeLog-1.9.3:82699:41: "developpers" is a misspelling of "developers"
doc/ChangeLog-1.9.3:88455:41: "crtical" is a misspelling of "critical"
doc/ChangeLog-1.9.3:92079:3: "overriden" is a misspelling of "overridden"
doc/ChangeLog-1.9.3:92619:32: "disbale" is a misspelling of "disable"
enc/trans/gb18030-tbl.rb:38377:4: "DAED" is a misspelling of "DEAD"
ext/win32ole/win32ole.c:489:12: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:497:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:498:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:499:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:500:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:501:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:502:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:503:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:504:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:505:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:506:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:507:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:508:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:509:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:510:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:511:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:512:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:513:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:514:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:515:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:516:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:517:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:518:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:519:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:520:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:521:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:522:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:523:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:524:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:525:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:526:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:527:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:528:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:529:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:530:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:531:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:532:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:533:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:534:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:535:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:536:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:537:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:538:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:539:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:540:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:541:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:542:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:543:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:544:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:545:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:546:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:547:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:548:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:549:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:550:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:551:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:552:8: "MACHING" is a misspelling of "MACHINE"
lib/rdoc/markdown/entities.rb:624:3: "andd" is a misspelling of "and"
lib/rdoc/markdown/entities.rb:1147:3: "infintie" is a misspelling of "infinite"
lib/rdoc/rd/inline_parser.rb:708:13: "eles" is a misspelling of "eels"
lib/rdoc/rd/inline_parser.rb:710:13: "eles" is a misspelling of "eels"
sample/trick2018/05-tompng/remarks.markdown:31:42: "apperance" is a misspelling of "appearance"
sample/trick2018/README.md:10:34: "dimentional" is a misspelling of "dimensional"
spec/ruby/CHANGES.before-2008-05-10:3738:12: "wierd" is a misspelling of "weird"
spec/ruby/CHANGES.before-2008-05-10:4228:4: "writen" is a misspelling of "written"
spec/ruby/CHANGES.before-2008-05-10:5370:45: "paramter" is a misspelling of "parameter"
spec/ruby/CHANGES.before-2008-05-10:9280:19: "moreso" is a misspelling of "more"
spec/ruby/CHANGES.before-2008-05-10:11888:30: "inheritence" is a misspelling of "inheritance"
spec/ruby/CHANGES.before-2008-05-10:12263:12: "paramter" is a misspelling of "parameter"
spec/ruby/CHANGES.before-2008-05-10:14818:4: "Commiting" is a misspelling of "Committing"
spec/ruby/CHANGES.before-2008-05-10:15115:18: "imcompatible" is a misspelling of "incompatible"
spec/ruby/CHANGES.before-2008-05-10:16700:271: "tyrany" is a misspelling of "tyranny"
spec/ruby/CHANGES.before-2008-05-10:17038:4: "Commiting" is a misspelling of "Committing"
spec/ruby/CHANGES.before-2008-05-10:17066:4: "Commiting" is a misspelling of "Committing"
spec/ruby/core/string/slice_spec.rb:157:57: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:216:51: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:229:51: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:248:55: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:51:53: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:62:50: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:192:52: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:266:58: "ther" is a misspelling of "there"
test/rexml/data/documentation.xml:410:30: "undesireable" is a misspelling of "undesirable"
test/rexml/data/much_ado.xml:4738:25: "throughly" is a misspelling of "thoroughly"
test/rexml/data/ofbiz-issues-full-177.xml:817:50: "occured" is a misspelling of "occurred"
test/rexml/data/ofbiz-issues-full-177.xml:3734:6: "facilites" is a misspelling of "facilities"
test/rexml/data/ofbiz-issues-full-177.xml:4775:10: "overriden" is a misspelling of "overridden"
test/rexml/data/ofbiz-issues-full-177.xml:7029:33: "compatiblity" is a misspelling of "compatibility"
```

## after

```
$ misspell .
configure.ac:2545:2: "nto" is a misspelling of "not"
configure.ac:2593:2: "nto" is a misspelling of "not"
doc/ChangeLog-0.60_to_1.1:541:3: "avaiable" is a misspelling of "available"
doc/ChangeLog-0.60_to_1.1:3576:20: "concatination" is a misspelling of "contamination"
doc/ChangeLog-2.1.0:16041:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.8.0:10128:29: "ASIGN" is a misspelling of "ASSIGN"
doc/ChangeLog-2.0.0:8597:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11772:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11790:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11825:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11836:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11845:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11857:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11889:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11974:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11976:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11976:47: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11979:43: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:11981:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12041:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12054:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12095:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12486:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12492:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12496:30: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12498:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12498:49: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12499:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12499:49: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:12508:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:15154:51: "infinit" is a misspelling of "infinite"
doc/ChangeLog-2.0.0:15399:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.0.0:21274:29: "execption" is a misspelling of "exception"
doc/ChangeLog-2.2.0:1005:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.4.0:891:51: "nonsence" is a misspelling of "nonsense"
doc/ChangeLog-2.4.0:9382:25: "addopt" is a misspelling of "adopt"
doc/ChangeLog-2.3.0:4294:51: "convertion" is a misspelling of "conversion"
doc/NEWS-1.9.2:243:36: "avaiable" is a misspelling of "available"
doc/ChangeLog-YARV:1484:29: "separete" is a misspelling of "seperate"
doc/ChangeLog-YARV:1484:38: "assersions" is a misspelling of "assertions"
doc/ChangeLog-YARV:4976:45: "arugment" is a misspelling of "argument"
doc/ChangeLog-YARV:6325:11: "analisys" is a misspelling of "analysis"
doc/ChangeLog-YARV:6492:1: "redesgin" is a misspelling of "redesign"
doc/NEWS-2.2.0:284:14: "addopt" is a misspelling of "adopt"
enc/Makefile.in:52:23: "DONIG" is a misspelling of "DOING"
enc/trans/big5-hkscs-tbl.rb:10399:4: "DAED" is a misspelling of "DEAD"
enc/trans/big5-hkscs-tbl.rb:28817:11: "DAED" is a misspelling of "DEAD"
enc/trans/big5-uao-tbl.rb:14114:4: "DAED" is a misspelling of "DEAD"
doc/ChangeLog-1.9.3:6960:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:7544:43: "embeded" is a misspelling of "embedded"
doc/ChangeLog-1.9.3:9566:37: "Assersions" is a misspelling of "Assertions"
doc/ChangeLog-1.9.3:9784:3: "funciton" is a misspelling of "function"
doc/ChangeLog-1.9.3:13493:39: "DONIG" is a misspelling of "DOING"
doc/ChangeLog-1.9.3:49252:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49252:46: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49267:16: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49276:19: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49280:26: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49282:43: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49290:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:49293:15: "addopt" is a misspelling of "adopt"
doc/ChangeLog-1.9.3:62972:41: "prototye" is a misspelling of "prototype"
doc/ChangeLog-1.9.3:76349:31: "arguemnts" is a misspelling of "arguments"
doc/ChangeLog-1.9.3:77288:3: "Execption" is a misspelling of "Exception"
doc/ChangeLog-1.9.3:77438:17: "initalize" is a misspelling of "initialize"
doc/ChangeLog-1.9.3:82699:41: "developpers" is a misspelling of "developers"
doc/ChangeLog-1.9.3:88455:41: "crtical" is a misspelling of "critical"
doc/ChangeLog-1.9.3:92079:3: "overriden" is a misspelling of "overridden"
doc/ChangeLog-1.9.3:92619:32: "disbale" is a misspelling of "disable"
enc/trans/euckr-tbl.rb:4920:4: "DAED" is a misspelling of "DEAD"
enc/trans/gb18030-tbl.rb:38377:4: "DAED" is a misspelling of "DEAD"
enc/trans/gbk-tbl.rb:19254:4: "DAED" is a misspelling of "DEAD"
ext/win32ole/win32ole.c:489:12: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:497:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:498:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:499:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:500:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:501:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:502:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:503:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:504:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:505:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:506:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:507:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:508:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:509:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:510:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:511:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:512:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:513:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:514:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:515:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:516:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:517:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:518:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:519:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:520:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:521:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:522:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:523:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:524:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:525:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:526:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:527:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:528:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:529:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:530:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:531:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:532:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:533:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:534:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:535:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:536:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:537:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:538:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:539:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:540:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:541:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:542:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:543:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:544:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:545:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:546:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:547:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:548:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:549:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:550:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:551:8: "MACHING" is a misspelling of "MACHINE"
ext/win32ole/win32ole.c:552:8: "MACHING" is a misspelling of "MACHINE"
lib/rdoc/markdown/entities.rb:624:3: "andd" is a misspelling of "and"
lib/rdoc/markdown/entities.rb:1147:3: "infintie" is a misspelling of "infinite"
lib/rdoc/rd/inline_parser.rb:708:13: "eles" is a misspelling of "eels"
lib/rdoc/rd/inline_parser.rb:710:13: "eles" is a misspelling of "eels"
sample/trick2018/05-tompng/remarks.markdown:31:42: "apperance" is a misspelling of "appearance"
sample/trick2018/README.md:10:34: "dimentional" is a misspelling of "dimensional"
spec/ruby/CHANGES.before-2008-05-10:3738:12: "wierd" is a misspelling of "weird"
spec/ruby/CHANGES.before-2008-05-10:4228:4: "writen" is a misspelling of "written"
spec/ruby/CHANGES.before-2008-05-10:5370:45: "paramter" is a misspelling of "parameter"
spec/ruby/CHANGES.before-2008-05-10:9280:19: "moreso" is a misspelling of "more"
spec/ruby/CHANGES.before-2008-05-10:11888:30: "inheritence" is a misspelling of "inheritance"
spec/ruby/CHANGES.before-2008-05-10:12263:12: "paramter" is a misspelling of "parameter"
spec/ruby/CHANGES.before-2008-05-10:14818:4: "Commiting" is a misspelling of "Committing"
spec/ruby/CHANGES.before-2008-05-10:15115:18: "imcompatible" is a misspelling of "incompatible"
spec/ruby/CHANGES.before-2008-05-10:16700:271: "tyrany" is a misspelling of "tyranny"
spec/ruby/CHANGES.before-2008-05-10:17038:4: "Commiting" is a misspelling of "Committing"
spec/ruby/CHANGES.before-2008-05-10:17066:4: "Commiting" is a misspelling of "Committing"
spec/ruby/core/string/slice_spec.rb:157:57: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:216:51: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:229:51: "ther" is a misspelling of "there"
spec/ruby/core/string/slice_spec.rb:248:55: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:51:53: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:62:50: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:192:52: "ther" is a misspelling of "there"
spec/ruby/core/string/shared/slice.rb:266:58: "ther" is a misspelling of "there"
test/rexml/data/documentation.xml:410:30: "undesireable" is a misspelling of "undesirable"
test/rexml/data/much_ado.xml:4738:25: "throughly" is a misspelling of "thoroughly"
test/rexml/data/ofbiz-issues-full-177.xml:817:50: "occured" is a misspelling of "occurred"
test/rexml/data/ofbiz-issues-full-177.xml:3734:6: "facilites" is a misspelling of "facilities"
test/rexml/data/ofbiz-issues-full-177.xml:4775:10: "overriden" is a misspelling of "overridden"
test/rexml/data/ofbiz-issues-full-177.xml:7029:33: "compatiblity" is a misspelling of "compatibility"
```

### CODE REVIEW

File: internal.h [hunk 1]
```diff
@@ -2035,7 +2035,7 @@ rb_pid_t rb_fork_async_signal_safe(int *status, int (*chfunc)(void*, char *, siz
 VALUE rb_execarg_new(int argc, const VALUE *argv, int accept_shell, int allow_exc_opt);
 struct rb_execarg *rb_execarg_get(VALUE execarg_obj); /* dangerous.  needs GC guard. */
 VALUE rb_execarg_init(int argc, const VALUE *argv, int accept_shell, VALUE execarg_obj, int allow_exc_opt);
-int rb_execarg_addopt(VALUE execarg_obj, VALUE key, VALUE val);
+int rb_execarg_adopt(VALUE execarg_obj, VALUE key, VALUE val);
```
Comment (mikegee) [inline internal.h]: Is it true that this change might break a gem with native extensions that calls `rb_execarg_addopt`?

Comment (seratch) [reply internal.h]: @mikegee 
Thanks. As I mentioned in the description of this PR, I am willing to revert the change if it is suspicious. I will do if the team is not 100% confident about the safety.

>Also, if the change on the method rb_execarg_addopt should not be included, I can revert the change (I judged the change is safe enough, though).

Comment (mikegee) [reply internal.h]: I totally read your description, I swear. 😃 

I have no idea if this is safe or not. I'm glad you investigated it 👍

**Comment by hsbt** [comment]
```
ccan/list/list.h
ext/nkf/nkf-utf8/config.h
spec/ruby/core/signal/signame_spec.rb
```

Above files are vendoring files. We shouldn't fix until fixing on upstream.

```
lib/rdoc/rd/block_parser.rb
lib/rdoc/rd/inline_parser.rb
```

Also, Above files are vendoring files and auto-generated code. Please fix on https://github.com/ruby/rdoc/ at first.

**Comment by seratch** [comment]
@hsbt Thank you for your response. I understand it. Sorry, I overlooked that a portion of the files I directly modified are automatically generated. 

Anyway, I understand that most of the changes should be done for upstream. Let me close this pull request. Thank you for your time.

File: internal.h (hunk 1 above)
Comment (nobu) [reply internal.h]: It stands for "add option", not single word "adopt", I think.

Comment (seratch) [reply internal.h]: I now understand that the name is not a typo. Thank you 🙇

**Pull request closed without merging**